ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -276,6 +276,9 @@ void RSkComponentTextInput::processEventKey (rnsKey eventKeyType,bool* stopPropa
           if(cursor_.locationFromEnd < cursor_.end ){
             RNS_LOG_DEBUG("[processEventKey]Left key pressed cursor_.locationFromEnd = "<<cursor_.locationFromEnd);
             cursor_.locationFromEnd++; // locationFromEnd
+            textInputMetrics.selectionRange.location = cursor_.end - cursor_.locationFromEnd;
+            textInputMetrics.selectionRange.length = 0;
+            textInputEventEmitter->onSelectionChange(textInputMetrics);
```

Comment:
1. What is the behavior if caretHidden prop is set 
2. Can we add comment on why length value is been set to 0 
3. Can we club common code of Left & Right keys since many LOC is duplicated for these two events.
4. Should the sequence be drawAndSubmit and then sent events ( because all other events are following this sequence) 
5. Please update the test app

Reply:
If caretHidden flag is set to true, the cursor won't be visible but still you can navigate to left or right. 
In both the cases either caretHidden flag is set to true or false, behavior is same but only difference is cannot able to see the cursor when flag set to true.